File: 백준/Bronze/15633. Fan Death/Fan Death.java
import java.util.Scanner;
import java.lang.Math;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int rlt=0,n = scan.nextInt();
        for(int i=1;i<n+1;i++) {
            if(n%i==0)
                rlt+=i;
        }
        System.out.print(rlt*5-24);
    }
}

File: 백준/Bronze/15633. Fan Death/Fan Death.c
#include <stdio.h>
int main(void) {
    int n,rlt=0;
    scanf("%d",&n);
    for(int i=1;i<n+1;i++) {
        if(n%i==0)
            rlt+=i;
    }
    printf("%d",rlt*5-24);
    return 0;
}